en/docs/manage/cloud/overview.md

```
@@ -0,0 +1,19 @@
+---
+title: "WSO2 Cloud"
+description: "Manage integrations deployed to WSO2 Cloud - Integration Platform, including lifecycle operations, configuration, and environment promotion."
+keywords: [wso2 integrator, wso2 cloud, manage, overview]
+---
+
+{/* TODO: Work in progress */}
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)